### DISCUSSION TITLE
Problem with a Plugin

### DISCUSSION THREAD

**Discussion by SnowYerry** [bootscore/bootscore#608] [Q&A]
I recently implemented this plugin, 'YITH Frontend Manager for WooCommerce'. It essentially allows managing orders from the frontend, including editing them, etc. My issue is that I can't edit them with the Bootscore theme. I know it's the problem because if I change the theme, I can edit them. In fact, the console sends me this error when I'm using the Bootscore theme: 'Uncaught ReferenceError: woocommerce_admin_meta_boxes is not defined at product_order/?id=7560'. I'd like to know if there's any way to solve it.

**Comment by crftwrk** [comment]
Well, we do not have this plugin and cannot check this issue. Of course, it's impossible to provide support for all 50000+ WP plugins out there and we did no changes in `woocommerce_admin_meta_boxes`. 

> I know it's the problem because if I change the theme, I can edit them.

The question is: Which theme? In this case, I suggest to install a similar Bootstrap 5 theme like [Understrap](https://github.com/understrap/understrap)  and check if plugin works with this one. If so, we can investigate further and check the code for the difference.

BTW, a link to your project to reproduce the issue is always very helpful ;-)

**Reply by SnowYerry** [reply]
The themes I was testing were Astra and Storefront. I also tried the one you mentioned, "Understrap," and it works fine for me to edit the products.

I'm leaving a link here that is a video showing the problem: 
https://drive.google.com/file/d/1TsocjqiuQ2eMACl1wBwPrnTAui60Q1PC/view?usp=share_link

This other link is for the plugin: 
***Admin edit:*** removed link as links to paid plugins aren't allowed.

**Reply by crftwrk** [reply]
Ok, then it seems there is something missing. Please activate the Understrap theme and try to locate the script. Open `inc/woocommerce.php` and delete each snippet until Frontend Manager plugin stops working. Do the same with each line in `functions.php`. If you found the missing snippet we can have a look what it does and add it to bootScore theme.

**Reply by SnowYerry** [reply]
Hi, I was finally able to do what you suggested, my apologies for the delay. I tried it, but it didn't work, so I started to investigate the error and the differences between the themes. I'll send you a video, basically the first one that appears is Understrap since it doesn't have the error and can be modified. The second one is Bootscore, and that's where the error is occurring.

I also checked the entire code of both themes and noticed that in Bootscore there's a script that is missing and it's called <script id='wc-admin-meta-boxes-js-extra'>, it's also shown in the video. I tried to find that script, but I couldn't figure out where it's located. I hope this information is useful for you to see what we can modify. Thank you very much in advance.

https://drive.google.com/file/d/10-UCGNq1Osz78_1tnAwLs53XMaVpHcGp/view?usp=sharing

**Reply by crftwrk** [reply]
Quickly checked the Understrap and didn't find this as well. But this lead me to another idea, maybe plugin does not work with the AJAX cart. Please disable AJAX cart and let me know if that helps. Code goes to child's `functions.php`:

```php
// Disable AJAX Cart
function register_ajax_cart() {
}
add_action('after_setup_theme', 'register_ajax_cart');
```

**Reply by SnowYerry** [reply]
I just did that, and no, it didn't work. I'm also noticing that maybe the problem lies in Bootscore not importing all the plugin scripts, as this one isn't found either.
<script src='https://papeleriastar.com/wp-content/plugins/yith-frontend-manager-for-woocommerce-premium/assets/js/yith-frontend-manager-media.min.js?ver=6.3.2' id='yith-frontend-manager-media-js-js'></script>
This might be triggering the error, which is strange because it does let me access the plugin's frontend page, but editing is a no-go, hahaha.

**Reply by crftwrk** [reply]
Maybe sharing a link to your project will help to clear it up ;-)

If it's a JS error, these files are loaded by the theme:

- https://github.com/bootscore/bootscore/blob/main/js/theme.js
- https://github.com/bootscore/bootscore/blob/main/js/lib/bootstrap.bundle.min.js
- https://github.com/bootscore/bootscore/blob/main/woocommerce/js/woocommerce.js
- https://github.com/bootscore/bootscore/blob/main/woocommerce/inc/ajax-add-to-cart.php

You can delete each file's content and check if its something there.

**Reply by SnowYerry** [reply]
I did what you asked with the files, but no, it still doesn't work.

I made a copy of the project; I didn't do it completely because it was going to take me more time, but with the little I did, it still doesn't work. So, look, this is the link to the project:
https://github.com/SnowYerry/OfficeStar-wordpress

And this is the link for the database:
https://drive.google.com/file/d/1k-0hluesSYlKYENSAb_9q2rPbnZgw1Pg/view?usp=sharing

I'm using XAMPP, and the username and password are the default ones, which means root with no password. To log in to WordPress, in case it asks, it doesn't matter since I don't have anything important there. Here is the username and password. ("I don't know how sharing projects in WordPress works, I'm new, hehe, but just in case.")
Yerry-OfficeStar-Backend
b)4!vZSBzLSOFcn21v

**Reply by crftwrk** [reply]
Sorry for my late reply, quite busy...

Thank you for your effort. But we do not have the time to setup your page and deeply check the bug. As written in our [guidelines](https://bootscore.me/documentation/support-contribute/), bug reports must include a link to a reduced test case provided by the user. Then we are able to view and debug direct in the browser, a description or a video is no help. If we have the reduced test case, we can guide you through to resolve it and when fixed you can create a pull request and we'll merge it here.
